examples/aiarena/example_bot.py

```diff
@@ -1,23 +0,0 @@
-import sc2
-
-class ExampleBot(sc2.BotAI):
-    def __init__(self):
-        # Improves bot performance by a little bit
-        self.raw_affects_selection = True
-        # The distance calculation method: 0 for raw python, 1 for scipy pdist, 2 for scipy cdist
-        self.distance_calculation_method = 2
```

Comment:
This is undocumented and should be somewhere visible, in my opinion, but I don't know where would be the best place to put it.
These are some performance improvements that can be enabled while the bot is active on the bot ladder servers, but can be annoying while developing bots.

Reply:
Maybe we can create extra documentation with "API Options" or something to document these?
I'll work something out and resubmit for your review.